shared/pkg/utils/trivy/sourcetypes.go

```diff
@@ -36,3 +42,60 @@ func KubeclaritySourceToTrivySource(sourceType utils.SourceType) (artifact.Targe
 	}
 	return artifact.TargetKind("Unknown"), fmt.Errorf("unable to convert source type %v to trivy type", sourceType)
 }
+
+// nolint:cyclop
+func UntarToDirectory(tarPath, destDirectory string) error {
```

Comment:
is it a trivy specific util func?

Reply:
Not particularly, although it is the only place its being used right now, I can move it out into a separate util

Reply:
Works! I'll include this in the next push